jobserver/management/commands/create_project_from_workspace.py

```diff
@@ -27,14 +27,12 @@ def _create_project(self, *, org, workspace):
         # generate a name from the slug
         name = " ".join(word.capitalize() for word in slug.split("-"))
 
-        project = Project.objects.create(org=org, name=name, slug=slug)
+        project = Project.objects.get_or_create(org=org, name=name, slug=slug)
 
         # tell the User what was made and where they can view it
         f = furl(settings.BASE_URL)
         f.path = project.get_absolute_url()
-        self.stdout.write(
-            f"Name: {project.name}\nSlug: {project.slug}\nURL:  ({f.url})"
-        )
+        self.stdout.write(f"Name: {project.name}\nURL:  {f.url}")
```

Comment:
Do we just not care enough about the slug?  Or is the url the same so it's repetitive?

Reply:
Both, I'm only outputting them so I can do a quick check they look useful and get to the created Project quickly, and realised the slug is right there in the URL…